src/course/syllabus/pre-app-7/schedule.md

```diff
@@ -0,0 +1,25 @@
+In this Wednesday's workshop we looked at [functions, callbacks and asynchronous JavaScript](https://learn.foundersandcoders.com/workshops/functions-callbacks-async/).
+
+## HTML Forms
+
+_60 minutes_
+
+Learn more about what happens when we submit HTML forms and how to collect their data by complete [this workshop](https://learn.foundersandcoders.com/workshops/html-forms/).
+
+## Promises
+
+_120 minutes_
+
+Learn about promises in [this workshop](https://learn.foundersandcoders.com/workshops/promise-practice/).
```

Comment:
This workshop was intended to be done after the `fetch` ones (so Part 2 has them fetching from the PokéAPI). If you wanted to do it this week I guess you could tell them not to worry about that part yet?